README.md

```diff
@@ -10,28 +10,51 @@
 
 ## Requirements
 
-* Node v8+
-
-> Recommended to use [NVM](https://github.com/creationix/nvm)
+- Node v8+
 
 ## Running the App
 
-* [Generate a GitHub personal access token](https://github.com/settings/tokens/new?scopes=&description=Hacktoberfest%20Checker) to ensure you don't get rate limited as often.
+- [Generate a GitHub personal access token](https://github.com/settings/tokens/new?scopes=&description=Hacktoberfest%20Checker) to ensure you don't get rate limited as often.
 
-* Create a `.env` file using `.env.example`. Or export your GitHub token as `GITHUB_TOKEN` environment variable in your system:
-   * Mac/Linux: `export GITHUB_TOKEN=YOUR_TOKEN`
-   * Windows (cmd.exe): `set GITHUB_TOKEN=YOUR TOKEN`
-   * Windows (PowerShell): `$env:GITHUB_TOKEN=YOUR TOKEN`
+- Create a `.env` file using `.env.example`.
+
+  ```bash
+  $ cp .env.example .env
+  ```
+
+- Alternatively, you cann export your GitHub token as `GITHUB_TOKEN` environment variable in your system:
+
+  - **Mac/Linux**
+
+    ```bash
+    export GITHUB_TOKEN=YOUR_TOKEN
+    ```
+
+  - **Windows**
+
+    For command prompt:
+
+    ```
+    set GITHUB_TOKEN=YOUR TOKEN
+    ```
+
+    For PowerShell:
+
+    ```
+    $env:GITHUB_TOKEN=YOUR TOKEN
+    ```
+
+- Install dependencies and start.
 
-* Install dependencies and run it.
   ```bash
   $ yarn
   $ yarn start
   ```
 
-* Open [http://localhost:5000](http://localhost:5000)
+- Open [http://localhost:5000](http://localhost:5000)
 
 Want to run the API server and the frontend in their own processes? Use this:
```

Comment:
There is no server in this fork. We can remove instructions for it as well

Reply:
Makes sense.